components/FloatingContact/src/stories/bem.stories.mdx

```diff
@@ -54,7 +54,7 @@ import { useEffect, useMemo } from "@storybook/client-api";
           class="denhaag-floating-contact-switch"
           aria-expanded="false"
           data-label-open="Contact"
-          data-label-close="Sluiten"
+          data-label-close="Contact"
```

Comment:
I added the data-attributes `data-label-open` and `data-label-close` to set the readable label.
If this shouldn't  be used, please set the label inline and remove the JS that sets the label as innerText/innerHTML.

Also I'm wondering, the label for screenreaders must stay `contact`, but what about the toggle label? Shouldn't that still be "Close/Sluiten" for deskop users (without screenreaders). That said, an aria-label is probably better to set? Not sure, please check with Accessibility Expert and Design.

Reply:
@paulAcato Fernands request: https://acato-nl.atlassian.net/browse/GDH-58?focusedCommentId=89613

Reply:
I know that, but that isn't what I asked.

Now you're appyling by JS a label thats already there.

What I'm asking is if it's possible with an aria-label or other attribute so that desktop users **without** screenreaders see a normal text like: "Close".

Otherwise the JS toggle can be removed with those data-attributes and set the label hardcoded.